platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/Log.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+enum Log {
+    static func error(_ message: String) {
+        print("Error: \(message)")
```

Comment:
You should use the Log.swift file from the PolyPod to have the logs actually written to logs file. Bringing it into core may be to much for this PR, I guess you could pass a Log function to Core.swift to be used to perform any logging.